packages/mermaid/src/defaultConfig.ts

```diff
@@ -253,6 +253,13 @@ const config: RequiredDeep<MermaidConfig> = {
     // TODO: can we make this default to `true` instead?
     useMaxWidth: false,
   },
+  packet: {
+    ...defaultConfigJson.packet,
+    useWidth: undefined,
+    // this is false, unlike every other diagram (other than gitGraph)
+    // TODO: can we make this default to `true` instead?
+    useMaxWidth: false,
```

Comment:
You're already using it as `true` :)

https://github.com/mermaid-js/mermaid/blob/136f1c50e725b6303c1f0bebec828be0e604ed97/packages/mermaid/src/diagrams/packet/renderer.ts#L18

Reply:
I'd recommend defaulting `useMaxWidth` to `true` too, to keep things consistent with every other diagram (other than `gitGraph`). That way we can delete this line.